Action.c

```diff
@@ -678,6 +702,9 @@ static const struct {
    { .key = "   F8 [: ", .roInactive = true,  .info = "lower priority (+ nice)" },
 #if (defined(HAVE_LIBHWLOC) || defined(HAVE_AFFINITY))
    { .key = "      a: ", .roInactive = true, .info = "set CPU affinity" },
+#endif
+#if defined(HAVE_BACKTRACE)
+   { .key = "      b: ", .roInactive = false, .info = "show the backtrace of user process" },
```

Comment:
Maybe keep this similar in style to existing entries.

```suggestion
   { .key = "      b: ", .roInactive = false, .info = "show process backtrace" },
```